esp/mlrs-wireless-bridge/mlrs-wireless-bridge.ino

```diff
@@ -386,7 +395,9 @@ if (g_protocol == WIRELESS_PROTOCOL_UDPCl) {
     // STA mode
     WiFi.mode(WIFI_STA);
     WiFi.disconnect();
+#ifdef FORCED_CLIENT_ADDRESS // Normally undefined to use DHCP
```

Comment:
I'd suggest to remove the comment here
and add a comment in the next line like // don't use DHCP but force to the specified IP